### PR TITLE
Create arscan.csl

### PR DESCRIPTION
**Pull request by avsz**
New style for the journal Les Cahiers d'ArScAn (ArScAn, CNRS)

### CODE REVIEW

File: archeologies-et-sciences-de-lantiquite.csl [hunk 1]
```diff
@@ -0,0 +1,700 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="fr-FR">
+  <info>
+    <title>Archéologies et Sciences de l’Antiquité</title>
+    <title-short>ArScAn</title-short>
+    <id>http://www.zotero.org/styles/archeologies-et-sciences-de-lantiquite</id>
+    <link href="http://www.zotero.org/styles/archeologies-et-sciences-de-lantiquite" rel="self"/>
```
Comment (rmzelle) [inline archeologies-et-sciences-de-lantiquite.csl]: @avsz, can you add a "documentation" link to the webpage/document describing this citation format? See https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link

File: archeologies-et-sciences-de-lantiquite.csl [hunk 2]
```diff
@@ -0,0 +1,700 @@
...
+    <author>
+      <name>AVSz</name>
+      <email>avszaba@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <summary>Le style ArScAn sert à la revue 'Les Cahiers d'ArScAn' et autres publications d'ArScAn (Archéologies et Sciences de l'Antiquité, CNRS), aux documents d'archéologie et d'histoire de l'art. Il est fait d'emprunts aux styles français du forum francophone de Zotero (invisu, etc.).</summary>
```
Comment (rmzelle) [inline archeologies-et-sciences-de-lantiquite.csl]: @avsz, is there a style in particular that you based this one on? If so, please refer to it via a "template" link. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

File: archeologies-et-sciences-de-lantiquite.csl [hunk 3]
```diff
@@ -0,0 +1,700 @@
...
+            <group delimiter=" " prefix=", ">
+              <text variable="genre"/>
+              <text term="presented at"/>
+              <text variable="event"/>
+            </group>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
```
Comment (rmzelle) [inline archeologies-et-sciences-de-lantiquite.csl]: @avsz, macros that aren't called anywhere aren't allowed. Could you delete the "event" (directly above) and "archive" macros (below at line 561)?

File: archeologies-et-sciences-de-lantiquite.csl [hunk 4]
```diff
@@ -0,0 +1,700 @@
...
+      </if>
+      <else>
+        <text term="cited" font-style="italic" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="translator">
+    <text term="translator" form="verb" suffix=" "/>
+    <names variable="translator">
+      <name font-style="normal" and="text" delimiter-precedes-last="never" et-al-min="15" initialize-with="." sort-separator=" ">
```
Comment (rmzelle) [inline archeologies-et-sciences-de-lantiquite.csl]: @avsz, every "et-al-min" attribute should be accompanied by an "et-al-use-first" attribute. ("et-al-min" specifies the minimum number of authors for which et-al abbreviation kicks in, and "et-al-use-first" specifies the number of authors that is still rendered before "et al.".

File: archeologies-et-sciences-de-lantiquite.csl [hunk 5]
```diff
@@ -0,0 +1,700 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="fr-FR">
+  <info>
+    <title>Archéologies et Sciences de l’Antiquité</title>
```
Comment (rmzelle) [inline archeologies-et-sciences-de-lantiquite.csl]: (@adam3smith, do you have any opinion whether we should put a hyphen after "l" in the style ID, for "l’Antiquité"? We're not consistent in the repository right now. Generated styles don't use a hyphen, while quite a few manually written styles do.

We're also not consistent about a preceding "d", as in e.g. "d'histoire". We do always forgo a hyphen for trailing "'s".)

Comment (rmzelle) [inline archeologies-et-sciences-de-lantiquite.csl]: (https://blog.tersmitten.nl/slugify/ doesn't place a hyphen for apostrophes, by the way)

Comment (adam3smith) [inline archeologies-et-sciences-de-lantiquite.csl]: I don't have much of an opinion; I can see the case for both. Given that, I'd go with what automated tools like slugify do and go without.

**Comment by rmzelle** [comment]
@avsz, if you set et-al settings on `<citation>` and `<bibliography>`, it's not necessary to put them on every `<name>` element, so I removed them. Let us know if you're happy with the style as it is. I also renamed the file name and title according to our standards here (e.g. "Archéologies et Sciences de l’Antiquité" instead of the abbreviated "ArScAn").

**Comment by avsz** [comment]
Thank you, your changes title and et-al settings) are fine.

Thank you very much for your help. I post a style on your repository for the first time. The process and your tools are really easy.

avsz


Envoyé depuis un mobile Samsung

-------- Message d'origine --------
De : "Rintze M. Zelle" <notifications@github.com> 
Date : 28/02/2017  6:48  (GMT+01:00) 
A : citation-style-language/styles <styles@noreply.github.com> 
Cc : avsz <ecureuilvio@gmail.com>,Mention <mention@noreply.github.com> 
Objet : Re: [citation-style-language/styles] Create arscan.csl (#2518) 
 
@avsz, if you set et-al settings on <citation> and <bibliography>, it's not necessary to put them on every <name> element, so I removed them. Let us know if you're happy with the style as it is. I also renamed the file name and title according to our standards here (e.g. "Archéologies et Sciences de l’Antiquité" instead of the abbreviated "ArScAn").

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**